.github/workflows/layers_govcloud.yml

```diff
@@ -0,0 +1,200 @@
+name: Layer Deployment (GovCloud)
+
+# GovCloud Layer Publish
+# ---
+# This workflow publishes a specific layer version in an AWS account based on the environment input.
+#
+# Using a matrix, we pull each architecture and python version of the layer and store them as artifacts
+# we upload them to each of the GovCloud AWS accounts.
+#
+# A number of safety checks are performed to ensure safety.
+
+on:
+  workflow_dispatch:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: choice
+        options:
+          - Gamma
+          - Prod
+        required: true
+      version:
+        description: Layer version to duplicate
+        type: string
+        required: true
+  workflow_call:
+    inputs:
+      environment:
+        description: Deployment environment
+        type: string
+        required: true
+      version:
+        description: Layer version to duplicate
+        type: string
+        required: true
+
+run-name: Layer Deployment (GovCloud) - ${{ inputs.environment }}
+
+permissions:
```

Comment:
This should be added to the other workflow as well, or it'll raise an OSSF scorecard issue

Reply:
why? the other workflow does not read antyhing, it just calls get-layer-version-by-arn

Reply:
Read the third paragraph here: https://github.com/ossf/scorecard/blob/main/docs/checks.md#token-permissions

Each workflow must set the default least level of permissions at the top level if it's also setting other permissions at the job level, which you're doing.

Reply:
if you don't set permission it's defaults to none, but I see the point of OSSF, not ideal. I will set it explicitly. 

We'd need to add it to python as well, since we copied the workflows.

Reply:
Yea